packaging/installer/methods/packages.md

```diff
@@ -1,6 +1,6 @@
 <!--
 title: "Install Netdata with .deb/.rpm packages"
-desciption: "Install the Netdata Agent with Linux packages that support Ubuntu, Debian, Fedora, RHEL, CentOS, openSUSE, and more."
+description: "Install the Netdata Agent with Linux packages that support Ubuntu, Debian, Fedora, RHEL, CentOS, openSUSE, and more."
```

Comment:
While this is in `<!--`, it's possible this metadata change may impact something...